examples/demo/01-snake/snake.c

```diff
@@ -334,7 +334,7 @@ SDL_AppResult SDL_AppEvent(void *appstate, SDL_Event *event)
     case SDL_EVENT_QUIT:
         return SDL_APP_SUCCESS;
     case SDL_EVENT_KEY_DOWN:
-        return handle_key_event_(ctx, event->key.scancode);
+        return (SDL_AppResult)handle_key_event_(ctx, event->key.scancode);
```

Comment:
No: You should change the return type of `handle_key_event_()` to `SDL_AppResult`, instead